ublox/src/ubx_packets/cfg_val.rs

```diff
@@ -1288,3 +1362,49 @@ pub enum TModePosType {
     /// Lat/Lon/Height position
     LLH = 1,
 }
+
+#[derive(Debug, Default, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum NavSpgFixMode {
+    /// 2D only
+    Only2D = 1,
+    /// 3D only
+    Only3D = 2,
+    /// Auto 2D/3D
+    #[default]
+    Auto = 3,
+}
+
+#[derive(Debug, Default, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum NavSpgUtcStandard {
```

Comment:
Hello,

no need to introduce, this already exists as `UtcStandardIdentifier` (used in other messages)